src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,68 @@
+package storage;
+
+import commandexceptions.JingHaoExceptions;
+import tasktype.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+public class Storage {
+    private static final String FILE_PATH = "data/tasklist.txt";
+
+    public static TaskList readFile() throws JingHaoExceptions, IOException{
+        TaskList currentList = new TaskList();
+        try{
+            File f = new File(FILE_PATH);
+            Scanner s = new Scanner(f);
+            while (s.hasNext()){
+                String currentLine = s.nextLine();
+                String[] informations = currentLine.split(",");
+                String taskType = informations[0];
+                switch (taskType){
+                case "T":
+                    Todo newTodo = new Todo(informations[2]);
+                    if(informations[1].equalsIgnoreCase("TRUE")){
+                        newTodo.check();
+                    }
+                    currentList.add(newTodo);
+                    break;
```

Comment:
Perhaps the cases can be abstracted into another method to shorten the method.

Reply:
Will do. Thanks!